install/resources/upgrade34.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * MyBB 1.8
+ * Copyright 2014 MyBB Group, All Rights Reserved
+ *
+ * Website: http://www.mybb.com
+ * License: http://www.mybb.com/about/license
+ *
+ */
+
+/**
+ * Upgrade Script: 1.8.5
+ */
+
+$upgrade_detail = array(
+	"revert_all_templates" => 0,
+	"revert_all_themes" => 0,
+	"revert_all_settings" => 0
+);
+
+@set_time_limit(0);
+
+function upgrade34_dbchanges()
+{
+	global $db, $output;
+
+	$query = $db->simple_select('settings', 'value', "name='username_method'");
```

Comment:
Why would I check a cached value instead of fetching it? It is an upgrade script not a production script.

I'm sure we do not check `$mybb->settings` when updating settings in any other version script.